modules/tinymce/src/core/main/ts/api/Annotator.ts

```diff
@@ -83,7 +85,13 @@ const Annotator = (editor: Editor): Annotator => {
     remove: (name: string): void => {
       const bookmark = editor.selection.getBookmark();
       identify(editor, Optional.some(name)).each(({ elements }) => {
-        Arr.each(elements, Remove.unwrap);
+        Arr.each(elements, (element) => {
+          if (isSpan(element)) {
+            Remove.unwrap(element);
+          } else {
+            removeDirectAnnotation(element);
+          }
+        });
```

Comment:
This appears to be duplicated with the below `removeAll` logic. Can we extract it out into a single function?

Reply:
I have refactored it out. Thanks for the suggestion